packages/supi/src/install/index.ts

```diff
@@ -403,20 +403,32 @@ export async function mutateModules (
       })
     }
 
+    function checkDependencyInPackage (dependencySet: Set<string>, inputs: string[]) {
+      let noneDependencyfound = true
+      for (const input of inputs) {
+        if (!dependencySet.has(input)) {
+          console.log(`No ${input} package found in dependencies of the project`)
+        } else {
+          noneDependencyfound = false
+        }
+      }
+      if (noneDependencyfound) {
+        throw new PnpmError('NO_PACKAGE_IN_DEPENDENCY', `No package found in dependencies of the project`)
```

Comment:
I'd suggest a more descriptive message like: None of the specified packages were found in the dependencies.